app/models/poster.rb

```diff
@@ -1,3 +1,7 @@
 class Poster < ApplicationRecord
-
+# !validations/relationships/data_manipulation happen here
+    def self.sort_by_creation(choice)
+        # *ActiveRecord method order that sorts the created_at attribute
+        order(created_at: choice.to_sym)
+    end
```

Comment:
I was just reading about this choice thing. Very cool. I wouldn't have thought to do this.